File: app/src/main/java/app/android/gifloaderviewmodel/model/Images.java
package app.android.gifloaderviewmodel.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Images {

    @SerializedName("fixed_height_downsampled")
    @Expose
    private Size fixedHeightDownsampled;

    public Size getFixedHeightDownsampled() {
        return fixedHeightDownsampled;
    }

    public void setFixedHeightDownsampled(Size fixedHeightDownsampled) {
        this.fixedHeightDownsampled = fixedHeightDownsampled;
    }
}

File: app/src/main/java/app/android/gifloaderviewmodel/model/Images.kt
package app.android.gifloaderviewmodel.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Images(
        @SerializedName("fixed_height_downsampled")
        @Expose
        var fixedHeightDownsampled: Size? = null
)
